Create itinerary page header

sakinko88 opened the pull request:
edit title form/ date/ destination button/ user icon/ member plus button/ swiper tab

sakinko88 committed:
changing the title

sakinko88 committed:
adding design and javascript

sakinko88 committed:
added swiper.js

sakinko88 committed:
added header items inside of create itinerary page

kredo-school commented on resources/views/Itinerary/create_itinerary_header.blade.php:
```diff
@@ -0,0 +1,210 @@
+
+@section('content')
+
+<style>
```
Please create separate css file

sakinko88 replied:
you mean, in style.css?

kredo-school replied:
you can name anything that could be related to what you are working but with .css extension and just import it in your blade file

kredo-school commented on resources/views/welcome.blade.php:
```diff
@@ -4,7 +4,7 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
-        <title>Laravel</title>
```
I think , we should remove "Laravel"

sakinko88 replied:
I removed it

kredo-school replied:
I mean we could still add <title> tag but use more helpful title that explains the page like your app name

sakinko88 committed:
Create_itinerary_page_header - made create itinerary UI

Changes:
- Adjusted itinerary _header_style
- Changed HTML
- Added return view to addList in itineraryController

sakinko88 committed:
Merge branch 'main' into Create_itinerary_page_header

# Conflicts:
#	app/Http/Controllers/ItinerariesController.php
#	resources/views/layouts/app.blade.php
#	routes/web.php
